test/pkg/test/framework.go

```diff
@@ -86,11 +88,8 @@ func NewFramework(ctx context.Context) *Framework {
 		TestCasesCreatedServiceNames:        make(map[string]bool),
 		TestCasesCreatedTargetGroupNames:    make(map[string]bool),
 	}
-
-	SetDefaultEventuallyTimeout(180 * time.Second)
+	SetDefaultEventuallyTimeout(5 * time.Minute)
 	SetDefaultEventuallyPollingInterval(10 * time.Second)
-	BeforeEach(func() { framework.ExpectToBeClean(ctx) })
-	AfterEach(func() { framework.ExpectToBeClean(ctx) })
 	return framework
```

Comment:
Moved  `BeforeEach()` and `AfterEach()` to the `test/suites/integration/suite_test.go`, will re-considering whether we really need it later.

Reply:
whats the problem of having it here?